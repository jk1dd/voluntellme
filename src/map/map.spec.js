import React from 'react'
import Map from './map'
import {shallow} from 'enzyme'

describe('<Map />', () => {
  it('should not crash', ()=> {
    const map = shallow(<Map />)
    expect(map).toBeTruthy
  })

  it('should have filler phrase', () => {
    const map = shallow(<Map />)
    expect(map.contains(<p>Look at organizations on the map!</p>)).toBeTruthy
  })
})
