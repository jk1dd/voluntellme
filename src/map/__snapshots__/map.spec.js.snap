// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> renders the right things 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <BrowserRouter>
    <div>
        <h1
            id="title"
        >
            VolunTellMe
        </h1>
        <ul
            id="navigation-bar"
        >
            <li>
                <Link
                    replace={false}
                    to="/"
                >
                    Info
                </Link>
            </li>
            <li>
                <Link
                    replace={false}
                    to="/table"
                >
                    Explore
                </Link>
            </li>
            <li>
                <Link
                    replace={false}
                    to="/map"
                >
                    Map
                </Link>
            </li>
        </ul>
        <Route
            component={[Function]}
            exact={true}
            path="/"
        />
        <Route
            component={[Function]}
            path="/table"
        />
        <Route
            component={[Function]}
            path="/map"
        />
    </div>
</BrowserRouter>,
  "nodes": Array [
    <BrowserRouter>
      <div>
            <h1
                  id="title"
            >
                  VolunTellMe
            </h1>
            <ul
                  id="navigation-bar"
            >
                  <li>
                        <Link
                              replace={false}
                              to="/"
                        >
                              Info
                        </Link>
                  </li>
                  <li>
                        <Link
                              replace={false}
                              to="/table"
                        >
                              Explore
                        </Link>
                  </li>
                  <li>
                        <Link
                              replace={false}
                              to="/map"
                        >
                              Map
                        </Link>
                  </li>
            </ul>
            <Route
                  component={[Function]}
                  exact={true}
                  path="/"
            />
            <Route
                  component={[Function]}
                  path="/table"
            />
            <Route
                  component={[Function]}
                  path="/map"
            />
      </div>
</BrowserRouter>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <App />,
      "_debugID": 5,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": App {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 3,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <BrowserRouter>
          <div>
                    <h1
                              id="title"
                    >
                              VolunTellMe
                    </h1>
                    <ul
                              id="navigation-bar"
                    >
                              <li>
                                        <Link
                                                  replace={false}
                                                  to="/"
                                        >
                                                  Info
                                        </Link>
                              </li>
                              <li>
                                        <Link
                                                  replace={false}
                                                  to="/table"
                                        >
                                                  Explore
                                        </Link>
                              </li>
                              <li>
                                        <Link
                                                  replace={false}
                                                  to="/map"
                                        >
                                                  Map
                                        </Link>
                              </li>
                    </ul>
                    <Route
                              component={[Function]}
                              exact={true}
                              path="/"
                    />
                    <Route
                              component={[Function]}
                              path="/table"
                    />
                    <Route
                              component={[Function]}
                              path="/map"
                    />
          </div>
</BrowserRouter>,
        "_debugID": 6,
        "_renderedOutput": <BrowserRouter>
          <div>
                    <h1
                              id="title"
                    >
                              VolunTellMe
                    </h1>
                    <ul
                              id="navigation-bar"
                    >
                              <li>
                                        <Link
                                                  replace={false}
                                                  to="/"
                                        >
                                                  Info
                                        </Link>
                              </li>
                              <li>
                                        <Link
                                                  replace={false}
                                                  to="/table"
                                        >
                                                  Explore
                                        </Link>
                              </li>
                              <li>
                                        <Link
                                                  replace={false}
                                                  to="/map"
                                        >
                                                  Map
                                        </Link>
                              </li>
                    </ul>
                    <Route
                              component={[Function]}
                              exact={true}
                              path="/"
                    />
                    <Route
                              component={[Function]}
                              path="/table"
                    />
                    <Route
                              component={[Function]}
                              path="/map"
                    />
          </div>
</BrowserRouter>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <App />,
}
`;
